src/commands/protocols/common/handle-protocol-message-command.js

```diff
@@ -60,7 +60,7 @@ class HandleProtocolMessageCommand extends Command {
             await this.blockchainModuleManager.getR2(blockchain),
             hashFunctionId,
             proximityScoreFunctionsPairId,
-            true,
+            false,
```

Comment:
what is the reasoning behind this change ? The sender node filters out offline nodes when determining the neighborhood, so when validating the neighborhood the same logic should be applied.

Reply:
With this change, if N nodes from the neighborhood are offline and "replaced" with N new nodes by the sender, these new nodes will return NACK because neighborhood validation will fail on their side.

Reply:
I have asked same question on other PR, you can find answer here: https://github.com/OriginTrail/ot-node/pull/2931#discussion_r1478193798